Project-I-Social-Card/social-card/src/components/HeaderComponents/HeaderContainer.js

```diff
@@ -1,2 +1,19 @@
 import React from 'react';
 import './Header.css';
+import ImageThumbnail from './ImageThumbnail';
+import HeaderTitle from './HeaderTitle';
+import HeaderContent from './HeaderContent';
+
+function HeaderContainer() {
+    return (
+        <div className = "HeaderContainer">
```

Comment:
It is better practice to use dashes instead of camelCase for *classNames*.
`className="header-container"`